src/proto/message/unparsed.rs

```diff
@@ -0,0 +1,40 @@
+//! Generic container for [`Extension`](super::Extension)-specific content
+
+use ssh_encoding::{self, Decode, Encode, Writer};
+
+/// Generic container for [`Extension`](super::Extension)-specific content.
+/// Accessing the inner `Vec<u8>` is only possible via conversion methods.
+#[derive(Debug, PartialEq, Clone)]
+pub struct Unparsed(Vec<u8>);
```

Comment:
This effectively reverts https://github.com/wiktor-k/ssh-agent-lib/pull/49

Reply:
furthermore: the `From` implementation allows you to place data in there formatted however you'd like, but you can't get it out :(

Reply:
🤦‍♂️ indeed. It's my mistake!

I was planning to add conversions from/to vectors and then did some other stuff and forgot about this.

Deeply sorry about it, I'll get this fixed and add some tests with non SSH messages as a way to repent for this blunder. 🙇‍♂️

Reply:
it's all good :) 

I opened https://github.com/wiktor-k/ssh-agent-lib/pull/72, I'd like to hear from both of you there.